src/libs/ReportActionsUtils.ts

```diff
@@ -681,7 +681,8 @@ function isActionableTrackExpense(reportAction: OnyxInputOrEntry<ReportAction>):
  *
  */
 function isResolvedActionTrackExpense(reportAction: OnyxEntry<ReportAction>): boolean {
-    const resolution = reportAction && 'resolution' in reportAction ? reportAction?.resolution : null;
+    const originalMessage = getOriginalMessage(reportAction);
+    const resolution = originalMessage && 'resolution' in originalMessage ? originalMessage?.resolution : null;
```

Comment:
Before accessing the 'resolution' in originalMessage, we should have checked the typeof originalMessage, and if it's an object, then we should have accessed the 'resolution'. Directly accessing the 'resolution' causes this [crash](https://github.com/Expensify/App/issues/45457) when we get the system message for exporting to CSV.